tools/tenscan/frontend/src/components/modules/rollups/rollup-details.tsx

```diff
@@ -0,0 +1,140 @@
+import TruncatedAddress from "../common/truncated-address";
+import KeyValueItem, { KeyValueList } from "@/src/components/ui/key-value";
+import { formatTimeAgo } from "@/src/lib/utils";
+import { Rollup } from "@/src/types/interfaces/RollupInterfaces";
+import Link from "next/link";
+
+export function RollupDetailsComponent({
+  rollupDetails,
+}: {
+  rollupDetails: Rollup;
+}) {
+  return (
+    <div className="space-y-8">
+      <KeyValueList>
+        <KeyValueItem label="ID" value={"#" + Number(rollupDetails?.ID)} />
+        <KeyValueItem
+          label="Timestamp"
+          value={formatTimeAgo(rollupDetails?.Timestamp)}
+        />
+        <KeyValueItem
+          label="Full Hash"
+          value={
+            <Link
+              href={`/rollup/${rollupDetails?.Hash}`}
+              className="text-primary"
+            >
+              <TruncatedAddress address={rollupDetails?.Hash} />
+            </Link>
+          }
+        />
+        <KeyValueItem
+          label="Rollup Header Hash"
+          value={
+            <Link
+              href={`/rollup/${rollupDetails?.Header?.hash}`}
+              className="text-primary"
+            >
+              <TruncatedAddress address={rollupDetails?.Header?.hash} />
+            </Link>
+          }
+        />
+        <KeyValueItem
+          label="L1 Hash"
+          value={<TruncatedAddress address={rollupDetails?.L1Hash} />}
+        />
+        <KeyValueItem
+          label="First Seq No."
```

Comment:
Use the same verbiage here ie "First batch sequence number" or "First Batch Seq. No"